### PR TITLE
Egg Hatching Automatically

### PR DESCRIPTION
**Pull request by cawk**
all credit goes to 
Bogenschuetze
made this as he didnt know how 
my code will be pulled aswell to allow users to make the unlimited incubators to only have 2k eggs when there are more than one incubator 
will be done in around 2-3 days when i return

### CODE REVIEW

File: PoGo.NecroBot.Logic/Tasks/UseAllIncubators.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                {
+                    var egg = incubator.ItemId == ItemId.ItemIncubatorBasicUnlimited
+                        ? unusedEggs.FirstOrDefault()
+                        : unusedEggs.LastOrDefault();
+
+                    if (egg == null)
+                        continue;
+
+                    var response = ctx.Client.Inventory.UseItemEggIncubator(incubator.Id, egg.Id).Result;
+                    Logger.Write($"Putting egg in incubator ({response.EggIncubator.TargetKmWalked - kmWalked:0.00}km left)");
```
Comment (yamashi) [inline PoGo.NecroBot.Logic/Tasks/UseAllIncubators.cs]: Please no Logger in Tasks, use machine.Fire and events

**Comment by Antidote78** [comment]
Tested ?

**Comment by Artud** [comment]
Fix conflicts.

**Comment by KurtCocain** [comment]
Is this in bot? Cause  dont haveany options for it and it doesn't incubate.

**Comment by cloudsstars** [comment]
@KurtCocain re download or use git  extensions to update

**Comment by Talamar1** [comment]
Shouldn't be in yet, code still has issues and that last commit is just a creation of a New Document.txt file which shouldn't be added into the PR

**Pull request closed without merging**